app/client/components/helpCentre/helpCentreArticle.tsx

```diff
@@ -7,6 +7,7 @@ import { captureException, captureMessage } from "@sentry/browser";
 import React, { useEffect, useState } from "react";
 import { minWidth } from "../../styles/breakpoints";
 import { trackEvent } from "../analytics";
+import { CallCentreEmailAndNumbers } from "../callCenterEmailAndNumbers";
```

Comment:
Is this the same module that the live chat button is being added to, i.e. will live chat also be present on every article page?

Reply:
Discussed, and should be fine